Refresh.GameServer/Configuration/GameServerConfig.cs

```diff
@@ -64,6 +64,21 @@ protected override void Migrate(int oldVer, dynamic oldConfig)
     /// Secondary safety switch incase the PSN and RPCN toggles somehow fail.
     /// </summary>
     public bool PermitAllLogins { get; set; } = true;
+
+    /// <summary>
+    /// Should all game logins be required to use Patchwork?
+    /// </summary>
+    public bool EnforcePatchwork { get; set; } = true;
+
+    /// <summary>
+    /// The minimum required major version of Patchwork on the client to be able to connect.
+    /// </summary>
+    public int RequiredPatchworkMajorVersion { get; set; } = 1;
+    
+    /// <summary>
+    /// The minimum required minor version of Patchwork on the client to be able to connect.
+    /// </summary>
+    public int RequiredPatchworkMinorVersion { get; set; } = 0;
```

Comment:
We would ideally add migrations to automatically increase this when Patchwork updates and increases security.

To avoid enforcing everyone to repatch all the time we should avoid increasing this whenever possible.